Anime-App/Feature/Anime/ViewModel/AnimeViewModel.swift

```diff
@@ -11,9 +11,9 @@ class AnimeViewModel {
         
     private(set) var animes: [Anime] = []
  
-    func getAnime(completion: @escaping (Result<Void,NetworkingError>) -> Void) {
-        // get request
-        let url = BaseUrl.baseUrl.url + UrlEndpoint.getAnime.endpoint
+    func getAnime(page: Int, offset: Int , completion: @escaping (Result<Void,NetworkingError>) -> Void) {
+        let url = BaseUrl.baseUrl.url + UrlEndpoint.getAnime.endpoint + "?" + UrlEndpoint.pagination.endpoint + "\(page)&page[offset]=\(offset)"
```

Comment:
Can you look into URLRequest and build up URLs using this and query params this is not a safe way to do this. It's better to use URLRequest and build up the query params etc.

Reply:
Also can you write a unit test for this to verify the URL is correct and looks like what you have in postman